TypeCobol.LanguageServer.Test/ProcessorTests/DataLayoutProcessorCSVTest.cs

```diff
@@ -17,14 +19,25 @@ public class DataLayoutProcessorCSVTest
         private const string ROOT_PATH = "ProcessorTests";
         private readonly DataLayoutProcessor _processor = new();
 
-        private static void DoTestProcessor(string sourceFileName, string expectedFileName, Func<CompilationUnit, (string Header, string[] Rows)> executeProcessor, [CallerMemberName] string testName = null)
+        private static void DoTestProcessor(string sourceFileName, string testDataFileName, Func<CompilationUnit, Position, (string Header, string[] Rows)> executeProcessor, bool isCopy = false, [CallerMemberName] string testName = null)
```

Comment:
For now the `executeProcessor` param has only one value, so it can be removed to simplify the signature.